sdk-tests/src/test/java/io/dapr/it/testcontainers/DaprContainerIT.java

```diff
@@ -129,9 +136,26 @@ public void testStateStore() throws Exception {
 
   @Test
   public void testPlacement() throws Exception {
```

Comment:
@salaboy I don't think this is required. I have ensured that DaprContainer uses a constant `IMAGE_TAG` and it is set to `1.14.1` for now. We can have a separate PR to upgrade to `1.15.1`.